packages/client/src/objectSet/createObjectSet.ts

```diff
@@ -141,11 +146,23 @@ export function createObjectSet<Q extends ObjectOrInterfaceDefinition>(
     asyncIter: async function*(): AsyncIterableIterator<Osdk<Q, "$all">> {
       let $nextPageToken: string | undefined = undefined;
       do {
-        const result = await base.fetchPage({ $nextPageToken });
```

Comment:
Was there a reason that we call `base.fetchPage` here instead of calling the unbound `fetchPageInternal`?